### DISCUSSION TITLE
Accessing an array in template

### DISCUSSION THREAD

**Discussion by Sparticuz** [twpayne/chezmoi#2239] [Q&A] [answered]
I'm trying to access a value of an array in a template. Specifically, a URI from Bitwarden's login variable. This is what I have right now
```
{{ (bitwarden "item" "the_item").login.uris[0].uri }}
```
However, it seems that [ is an invalid character
```
chezmoi: template: bin/executable_mountnas.tmpl:20: bad character U+005B '['
```

**Comment by twpayne** [comment, marked as answer]
For this you need the `index` function, for example:

    {{ (index (bitwarden "item" "the_item").login.uris 0).uri }}
